src/components/DaoCreator/index.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from "react";
+import { connect } from "react-redux";
+import { RouteComponentProps } from "react-router-dom";
+import DAOcreator from "@dorgtech/daocreator-ui-v1";
```

Comment:
so here is where we could use dynamic importing (lazy loading) to load this only at runtime, and reduce the size of the initial bundle: https://reactjs.org/docs/code-splitting.html